### DISCUSSION TITLE
How to change font style into italic.❓

### DISCUSSION THREAD

**Discussion by moraby256** [monkeytypegame/monkeytype#4570] [Questions] [answered]
Hello, I tried to change my **font style** to _italic_, but i didn't find font style in settings.
 
 Could you please help me with that ?
 
 i think to make it accessible to everyone by adding it into settings, that will be very helpful.

**Comment by Miodec** [comment, marked as answer]
Not possible at the moment. The only way would be to apply some custom css.
